include/astshim/Frame.h

```diff
@@ -265,8 +265,9 @@ class Frame : public Mapping {
     /**
     Compute a frameset that describes the conversion between this frame and another frame.
 
-    If conversion is possible, it returns a @ref FrameSet which describes the conversion and which may be used
-    (as a Mapping) to transform coordinate values in either direction.
+    If conversion is possible, it returns a shared pointer to a @ref FrameSet which describes
+    the conversion and which may be used (as a Mapping) to transform coordinate values in either direction.
+    Otherwise it returns an empty shared pointer.
```

Comment:
I agree. I'll add a flag or a new method. It should be a custom exception so it's easy to differentiate "not found" from other errors.